pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
Right, if you look at the code I posted, the class Sysfs is still subclass of driver. `version.rb` should _not_ have anything except for the `VERSION` constant. The version is usually defined in the module. The code above will work. You can have a module and class with the same name. There are tons of ruby projects that use this convention:

Here is how `RSpec` does it with their multiple gems:
- https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/version.rb
- https://github.com/rspec/rspec-expectations/blob/master/lib/rspec/expectations/version.rb
- https://github.com/rspec/rspec-rails/blob/master/lib/rspec/rails/version.rb
Take note how in each gem they require the other core gems.

And here is `Jekyll`:
- https://github.com/jekyll/jekyll/blob/master/lib/jekyll/version.rb
- https://github.com/jekyll/jekyll-compose/blob/master/lib/jekyll-compose/version.rb
- https://github.com/jekyll/jekyll-assets/blob/master/lib/jekyll/assets/version.rb
In this one take not how jekyll-assets imports Jekyll as well in `jekyll/assets.rb`